Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -881,6 +881,9 @@ public static void SetClientReady(NetworkConnection conn)
 
             // set ready
             conn.isReady = true;
+
+            if (conn.playerController != null)
+                SpawnObserversForConnection(conn);
```

Comment:
interesting. can you explain this further please? why should we do it and why is it bad that we didn't do it before?

Reply:
I've no idea when observers stopped working with scene changes, but clearly from the repro project I submitted in #978 and the one submitted in #974 it's not working and this seemed like the logical place to fix it.  If you've got a better solution, that's fine, but this definitely needs fixed one way or another.

Reply:
so this did work before, but doesn't work anymore now?
if you have time this week, could you maybe try to bisect so we know exactly what caused it? perhaps we introduced a bug elsewhere that we shouldn't

Reply:
I have no idea if or when it ever worked.

Reply:
I while ago I sent a PR that eliminated allocations rebuilding the observer list. I'm willing to bet if it broke it was there.

Reply:
@paulpach Could you please review your PR and see if you can un-break it or determine if the solution I propose here would be the correct thing to do?